### ISSUE TITLE
Log files for training

### ISSUE THREAD

**Issue by xychen2022** [ibrahimethemhamamci/CT-CLIP#25]
Hi, thanks for sharing the massive dataset, your codes, pretrained weights etc. For facilitate follow-ups, it would be helpful if you could share your training log files. Do you have plan to share these? Thanks

**Comment by sezginerr** [comment]
Hi @xychen2022, yes I can send the log files (ideally the updated trainings we will shortly push as it has been happened a while since the previous version and I am not sure if we still have the stdout files from the jobs), but it only shows the loss function which does not really provide any information except it goes lower.
